legacy_tests/nnbd_opted_in_with_optout/mono_pkg.yaml

```diff
@@ -0,0 +1,18 @@
+# See https://pub.dev/packages/mono_repo
+
+sdk:
+- pubspec
+- stable # We will want to hardcode 2.19 once 3.x comes out
```

Comment:
I'm not sure we need to run on stable at all. Running on 2.18 from the pubspec version should be enough.

Reply:
SGTM, swapped all these to run on 2.18